api/v2.0/swagger.yaml

```diff
@@ -229,6 +241,39 @@ paths:
           $ref: '#/responses/404'
         '500':
           $ref: '#/responses/500'
+  /projects/{project_name}/repositories/{repository_name}/artifacts/{reference}/additions/{addition}:
+    get:
+      summary: Get the addition of the specific artifact
+      description: Get the addition of the artifact specified by the reference under the project and repository. The addition can be "BUILD_HISTORY" for images and "VALUES.YAML", "README", "DEPENDENCIES" for charts
+      tags:
+        - artifact
+      operationId: getAddition
+      parameters:
+        - $ref: '#/parameters/requestId'
+        - $ref: '#/parameters/projectName'
+        - $ref: '#/parameters/repositoryName'
+        - $ref: '#/parameters/reference'
+        - name: addition
+          in: path
+          description: The addition name, "BUILD_HISTORY" for images; "VALUES.YAML", "README", "DEPENDENCIES" for charts
+          type: string
+          enum: [BUILD_HISTORY, VALUES.YAML, README, DEPENDENCIES]
```

Comment:
These UPPER words are in the URL? Maybe lower cases are better and consistent with the other path segments.

Backend const values can be different from the query parameters.